src/ontology/mondo-edit.obo

```diff
@@ -155551,10 +155550,7 @@ xref: SCTID:700211007 {source="MONDO:equivalentTo"}
 xref: UMLS:C1866994 {source="Orphanet:3138", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:3138/e", source="DOID:0060614", source="OMIM:181450"}
 is_a: MONDO:0015160 {source="Orphanet:3138"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
 is_a: MONDO:0015246 {source="Orphanet:3138"} ! syndromic anorectal malformation
-is_a: MONDO:0015853 {source="Orphanet:3138"} ! deficient breast volume or number
-is_a: MONDO:0018234 ! dysostosis
 is_a: MONDO:0018454 {source="MONDO:0018455-obsoleted"} ! dysostosis of genetic origin
-is_a: MONDO:0019054 ! congenital limb malformation
```

Comment:
was this manually removed by curator and therefore an intended change?

Reply:
same as above, I removed it